docs/index.md

```diff
@@ -7,106 +7,84 @@ This is the documentation for the EPCC SAFE.
     This document is currently under development and may be incomplete and/or misleading in places.
     Please continue to refer to the <a href="http://www.archer.ac.uk/documentation/safe-guide/">existing documentation</a> until this work is completed and this warning removed.
 
+[SAFE](https://www.archer.ac.uk/safe) is an online ARCHER2 service management system. This is new documentation for the latest version of SAFE version 4. 
 
 
+## [SAFE Guide for Individual Users](safe-for-users)
 
+Through [SAFE](https://www.archer.ac.uk/safe), individual users can request machine accounts, reset passwords, see available resources and track their usage. All users must be registered on SAFE before they can apply for their machine account. 
 
-<p><a href="https://www.archer.ac.uk/safe">SAFE</a> is an online ARCHER2 service management system. This is new documentation for the latest version of SAFE version 4. </p>
+Go to the [SAFE guide for individual users](safe-for-users)
 
-<h2><a href="safe-for-users">SAFE Guide for Individual Users</a></h2>
-<p>
-  Through SAFE, individual users can request machine accounts, reset passwords, see available resources 
-  and track their usage. All users must be registered on SAFE before they can apply for their machine account. 
-</p>
-<p>
-  Go to the <a href="safe-for-users">SAFE guide for individual users</a>
-</p>
+### [Registering, logging in, passwords](safe-for-users#reg-log-pass)
 
-<!-- used to generate sidebar TOC -->
-<div style="display:none;">
-<h2>Contents</h2>
-</div>
 
+- [How to register on SAFE](safe-for-users#register)
+- [How to login to SAFE](safe-for-users#login)
+- [How to use 2-factor authentication to SAFE](safe-for-users#2fac)
+- [How to change your personal details on SAFE](safe-for-users#details)
+- [How to change your email address on SAFE](safe-for-users#chemail)
+- [How to change your SAFE password](safe-for-users#chpass)
+- [How to reset your SAFE password if lost or forgotten](safe-for-users#reset)
+- [How to request a machine account](safe-for-users#getac)
+- [How to request access to a guest budget](safe-for-users#accguest)
+- [How to reset a password on your machine account](safe-for-users#reset_machine)
+- [How can to pick up your password for the service machine](safe-for-users#getpass)
+- [How to request access to a Package Group](safe-for-users#package-group)
```

Comment:
I am not sure we need this list any more, given that the interface will provide a menu with the list itself.